### PR TITLE
[TG-80] [feat] implemented adding and editing scenario

### PR DESCRIPTION
**Pull request by quasiunafantasia**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/garage-it/SmartHouse-frontend/pull/15" target="_blank">ReviewNinja

File: test/MockPromise.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+export class MockPromise {
```
Comment (quasiunafantasia) [inline test/MockPromise.js:1]: this guy will go away after we add a library for mocking promises

File: src/app/scenarios/create-scenario/createScenario.component.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import {RouteParams, Router} from 'angular2/router';
```
Comment (kosich) [inline src/app/scenarios/create-scenario/createScenario.component.js]: follow dash-n-dot naming convention
this file should've been named `screate-scenario.component.js`
see https://angular.io/styleguide#!#naming for details

Comment (quasiunafantasia) [inline src/app/scenarios/create-scenario/createScenario.component.js]: fixed in 3eb15a4

**Pull request merged**